src/app/games/conspirateurs/ConspirateursMove.ts

```diff
@@ -1,22 +1,21 @@
 import { Coord } from 'src/app/jscaip/Coord';
 import { Direction } from 'src/app/jscaip/Direction';
-import { MoveEncoder } from 'src/app/utils/Encoder';
+import { Encoder } from 'src/app/utils/Encoder';
 import { Move } from 'src/app/jscaip/Move';
 import { MoveCoord } from 'src/app/jscaip/MoveCoord';
 import { MoveCoordToCoord } from 'src/app/jscaip/MoveCoordToCoord';
 import { ArrayUtils } from 'src/app/utils/ArrayUtils';
 import { MGPFallible } from 'src/app/utils/MGPFallible';
 import { MGPSet } from 'src/app/utils/MGPSet';
 import { CoordSet } from 'src/app/utils/OptimizedSet';
-import { JSONValue, JSONValueWithoutArray, Utils } from 'src/app/utils/utils';
-import { assert } from 'src/app/utils/assert';
 import { ConspirateursFailure } from './ConspirateursFailure';
 import { ConspirateursState } from './ConspirateursState';
 import { MoveWithTwoCoords } from 'src/app/jscaip/MoveWithTwoCoords';
+import { Utils } from 'src/app/utils/utils';
 
 export class ConspirateursMoveDrop extends MoveCoord {
-    public static encoder: MoveEncoder<ConspirateursMoveDrop> =
-        MoveCoord.getFallibleEncoder(ConspirateursMoveDrop.from);
+
+    public static encoder: Encoder<ConspirateursMoveDrop> = MoveCoord.getFallibleEncoder(ConspirateursMoveDrop.from);
 
     public static from(coord: Coord): MGPFallible<ConspirateursMoveDrop> {
```

Comment:
tu pourrais en faire un of lui

Reply:
Parce que son .failure est pas traduit ?

Reply:
ouais, son failure est impossible à afficher, ça peut donc devenir un assert